single_tech_samples/databricks/project_lumus/README.md

```diff
@@ -17,36 +17,54 @@
 
 ## Milestones
 
+<a href = "https://github.com/Azure-Samples/modern-data-warehouse-dataops/projects/15">Project Board</a>
```

Comment:
Please remove HTML Markups from Readme 
You can use https://docs.github.com/en/github/writing-on-github/organizing-information-with-tables#creating-a-table instead